scripts/HOWTO-distribution.md

```diff
@@ -177,7 +177,7 @@ We roll some general code maintenance items into the release process.
         ant
 ```
 
-- This is a good place to make sure CATS still builds, see the [doc page](http://jmri.org/help/en/html/doc/Technical/CATS.shtml) - note that CATS has not been updated to compile cleanly with JMRI 4.*
+- This is a good place to make sure CATS still builds, see the [doc page](https://www.jmri.org/help/en/html/doc/Technical/CATS.shtml) - note that CATS has not been updated to compile cleanly with JMRI 4.*
```

Comment:
http://cats4ctc.wikidot.com/main:downloads
It says that release 2043 of CATS and designer package is compatible with JMRI 4.12 through current
See downloads on the bottom of the page.